app/ui/src/app/dashboard/dashboard_integrations/dashboard-integrations.component.ts

```diff
@@ -1,53 +1,73 @@
-import { Component, Input, ViewChild, OnChanges } from '@angular/core';
+import { Component, Input, ViewChild, OnInit } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
 import { Observable } from 'rxjs/Observable';
 
-import { log, getCategory } from '@syndesis/ui/logging';
-import { Connection, Connections, IntegrationOverview, IntegrationOverviews } from '@syndesis/ui/platform';
+import { DonutComponent, DonutConfig } from 'patternfly-ng';
 
-const category = getCategory('Dashboard');
+import { log } from '@syndesis/ui/logging';
+import {
+  Connection,
+  Connections,
+  IntegrationOverview,
+  IntegrationOverviews,
+  IntegrationSupportService
+} from '@syndesis/ui/platform';
 
 @Component({
   selector: 'syndesis-dashboard-integrations',
   templateUrl: './dashboard-integrations.component.html',
   styleUrls: ['./dashboard-integrations.component.scss']
 })
-export class DashboardIntegrationsComponent implements OnChanges {
-  chartData: number[];
-  @Input() integrations: IntegrationOverviews;
+export class DashboardIntegrationsComponent implements OnInit {
+
+  integrationsObserver: Observable<IntegrationOverviews>;
```

Comment:
`integrationsObserver` conveys the idea that this member is an `Observer` and as such can emit new values (as if it was a `Subject` or `BehaviorSubject` type).

Perhaps `integrationOverviews$` is a more expressive name for this field, since it clearly conveys both the type it represents and its observable nature.